rmidevice/rmidevice.cpp

```diff
@@ -303,10 +305,17 @@ int RMIDevice::ScanPDT(int endFunc, int endPage)
 		SetRMIPage(page);
 
 		for (addr = pdt_start; addr >= pdt_end; addr -= RMI_DEVICE_PDT_ENTRY_SIZE) {
-			rc = Read(addr, entry, RMI_DEVICE_PDT_ENTRY_SIZE);
+Retry:		rc = Read(addr, entry, RMI_DEVICE_PDT_ENTRY_SIZE);
```

Comment:
Do we need to reset the count after a successful read, or just allow up to 3 total failures?